src/NATS.Client.JetStream/NatsJSMsg.cs

```diff
@@ -6,17 +6,113 @@
 
 namespace NATS.Client.JetStream;
 
+public interface INatsJSMsg<out T>
```

Comment:
Same as `INatsMsg<T>`, we should annotate this type for awareness.

Reply:
added a few lines. please feel free to suggest edit or rewrite